tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -346,4 +346,30 @@ jobs:
           mergeTestResults: false
         condition: succeededOrFailed()
 
+      # Check Published Tests
```

Comment:
rather than duplicate these steps, i'd pull them out into a template (perhaps `log-test-failures.yml` or `include-log-test-failures.yml`, not sure why we have the include naming convention tbh but maybe best to follow it).